Makefile.in

```diff
@@ -20,7 +20,12 @@ EXT_LIB = @EXT_LIB@
 EXT_OBJ = @EXT_OBJ@
 EXT_DLL = @EXT_DLL@
 
+BIN_ANNOT = $(shell expr `echo $OCAMLVERSION | cut -f1 -d.` \>= 4)
```

Comment:
I don't see how this can possibly work. This test is always true.
Please use configure and define OCAML_HAS_BIN_ANNOT feature there.